gen/calc.py

```diff
@@ -339,6 +339,33 @@ def validate_bootstrap_tmp_dir(bootstrap_tmp_dir):
         "Must be an absolute path to a directory, although leave off the `/` at the beginning and end."
 
 
+def calculate_minuteman_min_named_ip_str(minuteman_min_named_ip):
+    return calculate_ip_erlstr(minuteman_min_named_ip)
+
+
+def calculate_minuteman_max_named_ip_str(minuteman_max_named_ip):
+    return calculate_ip_erlstr(minuteman_max_named_ip)
+
+
+def calculate_ip_erlstr(ip):
+    return '{' + ip.replace('.', ',') + '}'
```

Comment:
Probably not the best idea to call this "calculate_", probably something more like ip_to_erltuple?